# This message describ the grasp server request

# The global PointCloud
sensor_msgs/PointCloud2 global_cloud

# Coordinates of the object bounding box
manipulation_pepper/BoundingBoxCoord bounding_box